app/src/main/java/model/objects/movingObject/Creature.java

```diff
@@ -1,10 +1,36 @@
 package model.objects.movingObject;
 
+import lombok.Getter;
+import lombok.Setter;
+import tools.Vector2D;
+
 import java.awt.*;
+import java.io.Serializable;
 
 /*
  * Basic object of all objects in project
  */
-public class Creature {
+@Getter
+public class Creature implements Serializable {
+    private static Integer next_id = 0;
+
+    @Setter
+    protected boolean isVisible = true;
```

Comment:
po co isVisible skoro nie jest uzywana prawie ?

Reply:
ta i wyglada na to ze wszyscy zawsze beda visible

Reply:
isVisible się przyda, bo łatwiej jest ukryć ludzika jak wchodzi do windy, niż przesuwać ich razem z nią